README.md

```diff
@@ -16,148 +18,194 @@
+- Ionic - an open-source UI toolkit (version 6.0.3) 
 
-### Deploying The Daily Check App
+- Angular – a TypeScript-based, free and open-source web application framework (version 13) 
 
-After cloning this repo, install all the dependency modules by running `npm install`
+- Capacitor - Community/Electron - (version 4.1.0)  
 
-To run the app the application in a browser, run `ionic serve`
+- Electron - a framework for building desktop applications using JavaScript, HTML and CSS  
 
-To run the application directly during development without creating an exe, run the following command from the electron folder `npm run electron:start-live`
+<h3><a id="giga-meter-repos" class="anchor" aria-hidden="true" href="#giga-meter-repos"><svg class="octicon octicon-link" align="center" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"></path></svg></a>
+Github Repositories of GigaMeter </h3>
 
-To create a build from ionic, for dev environment run:
+- [GigaMeter Frontend](https://github.com/unicef/project-connect-daily-check-app)
+- [GigaMeter Backend](https://github.com/unicef/giga-meter-backend)
+- [Giga – documentation](#)
 
-```
-ionic build
-```
 
-OR, for production environment:
+<h2><a id="getting-started" class="anchor" aria-hidden="true" href="#getting-started"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"></path></svg></a>
+Getting Started</h2>
 
-```
-ionic build --prod 
-```
+### Setup and Running
+#### Installation 
 
-After running this command the build will be generated. To transfer the build to electron run the following command:
+- Download the latest version from the GigaMeter website.  
 
-```
-npx cap sync @capacitor-community/electron
-```
+    - The download will start automatically.  
 
-After that, navigate to the electron folder
+    - When the download is complete, double-click on the file – this usually appears on the bottom bar of your browser, otherwise, you will find the file in your ‘Downloads’ folder.  
 
-```
-cd .\electron\
-```
+    -  The installation process is straightforward, follow the wizard by clicking ‘Next’ on each step.  
 
-and create the windows build (.exe or msi) by runing the following command:
+- After successfully installing the app, it will send schools' internet connectivity information to Project Connect twice a day. 
 
-```
-npm run electron:make
-```
+#### Development Commands 
 
-The windows build can be found inside `/electron/dist/folder`
+<b>Basic development commands</b> 
 
-## Features
-* Checks school internet connection using minimal data (obtains metrics such as upload and download speed, latency, IP address, etc)
-* Automatically detects the country of a user 
-* Uploads results to a public database
+- After cloning the code from the GitHub repository https://github.com/unicef/project-connect-daily-check-app, install all the dependencies modules by running this command: npm install 
```

Comment:
ex:

- After cloning the code from the GitHub repository https://github.com/unicef/project-connect-daily-check-app, install all the dependencies modules by running this command: 
```
npm install 
```

Reply:
Noted, please check the edits and tell me if all is in par